### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by jbrask** [bordaigorl/rmview#29] [Q&A] [answered]
Thank you for making this available - it looks great and seems to be exactly what I have been looking for!

I'd like to try it out, but I have a couple of questions regarding installation, which was not clear to me from the instructions:

How do I install rmview, given that I am using Anaconda?

I have installed the dependencies  (PyQt5, Paramiko, Twisted) using conda, and I also managed to install rM-vnc-server and mxc_epdc_fb_damage on the reMarkable. I am running Ubuntu 18.04.

Also, when using pip, from where should the command "pip install ." be run? This makes no reference to rmview, so I presume it must be run from inside a folder with the downloaded source or something.

Any guidance is much appreciated!

**Comment by bordaigorl** [comment, marked as answer]
If you installed the dependencies through conda I think you should ignore the instructions about pip.
In principle, if everything worked, you should be able to run the script as any other script.
Since this is a package, go to the `src` folder and execute `python -m rmview`. You should see a window and be prompted for parameters.

**Reply by jbrask** [reply]
Thanks a lot for the quick reply!

I downloaded the zip from githup and extracted it. When I run `python -m rmview` from within the `src` folder however, I get the following error message:

> Traceback (most recent call last):
  File "/home/jonatan/anaconda3/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/jonatan/anaconda3/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/jonatan/Downloads/rmview-vnc/src/rmview/__main__.py", line 1, in <module>
    from .rmview import *
  File "/home/jonatan/Downloads/rmview-vnc/src/rmview/rmview.py", line 5, in <module>
    from . import resources
ImportError: cannot import name 'resources' from 'rmview' (/home/jonatan/Downloads/rmview-vnc/src/rmview/__init__.py)

**Reply by jbrask** [reply]
I now ran the pip install (`pip install .` from the root folder of the extracted zip), and it works :).

Running `rmview` it prompts me for IP and password and then it runs just fine. Very fast and smooth, and the red pen pointer feature is great.

![thanks](https://user-images.githubusercontent.com/5961328/101960959-edaee880-3c08-11eb-81ad-fa5b042344c9.png)

**Reply by bordaigorl** [reply]
Ah great! I'll add instructions for Anaconda users then! Thanks for reporting back.
I just have no experience with Anaconda, I just realised you can still use pip with it, so indeed the `pip install .` makes it work.
[If you are curious: the resources.py module gets generated from the graphic files; pip install generates it for you, otherwise you have to generate it manually with pyrcc5 (see setup.py for details)]
